test/device-unit-tests.js

```diff
@@ -82,7 +82,7 @@ describe( "device class unit tests", function() {
             ); 
       });
    });
-   describe( "device is instantiated with no public key", function() {
+   describe( "device is instantiated with no private key", function() {
 //
 // Verify that the device module throws an exception when there is
 // no valid public key file.
```

Comment:
Can you update the comment (public to private) as well?

Reply:
OK

Reply:
Just to clarify I meant the comment on line 88
 `// no valid public key file.`
 change to 
 `// no valid private key file.`